File: messageDotNet/grpc/EventData/ServiceUnregister.cs
using System;
using UnityEngine;

/// <summary>
/// service 销毁事件
/// </summary>
[Serializable]
public class ServiceUnregisterEvent
{
    // 消息版本号 值为毫秒时间戳
    public long MsgVersion;
    public ServiceData Service; 
}

File: messageGo/grpc/pubsubEventData/service_unregister.go
package pubsubEventData

import (
	base_data "game-message-core/grpc/baseData"
)

type ServiceUnregisterEvent struct {
	MsgVersion int64                 `json:"msgVersion"` // 消息版本号 值为毫秒时间戳
	Service    base_data.ServiceData `json:"service"`    //
}

func (p *ServiceUnregisterEvent) Clear() {
	p.MsgVersion = 0
	p.Service.Clear()
}
